finance-service/src/main/java/com/school/finance/domain/StudentPayment.java

```diff
@@ -0,0 +1,21 @@
+package com.school.finance.domain;
```

Comment:
why do you add this entity, in this issue we just want to get factor and do not need to do any thing about payment!
So you do not use this entity in any controller so why do you add this?